pkg/microservice/aslan/core/service/handler/service.go

```diff
@@ -39,14 +39,7 @@ func ListServiceTemplate(c *gin.Context) {
 	ctx := internalhandler.NewContext(c)
 	defer func() { internalhandler.JSONResponse(c, ctx) }()
 
-	var tmpResp *commonservice.ServiceTmplResp
-	tmpResp, ctx.Err = commonservice.ListServiceTemplate(c.Query("productName"), ctx.Logger)
-
-	// 如果是数据统计页面的请求，则需要将托管环境的服务也返回
-	if c.Query("requestFrom") == "stat" {
-		svcservice.ListServicesInExtenalEnv(tmpResp, ctx.Logger)
-	}
-	ctx.Resp = tmpResp
+	ctx.Resp, ctx.Err = commonservice.ListServiceTemplate(c.Query("productName"), c.Query("envName"), ctx.Logger)
```

Comment:
we don't need a envName when querying service templates, it makes no sense

Reply:
leave it later

Reply:
new router